### PR TITLE
edits and many responses

### PR DESCRIPTION
**Pull request by petrelharp**
I've made a pass through the manuscript and the responses to reviewers. The main thing that we still need to do is to add in the figures describing SLiM and some text discussing the results about rescaling - so, waiting on #150. We are also waiting on #160 for census sizes in figures 2 and 3.

Attached is a latexdiff to the previous HEAD (@ckyriazis's edits); also note that now you can run `make popSimManu-diff-to-submission.pdf` to make a highlighted diff to the originally submitted version.

In this PR I'm:

- closing #164 
- closing #165 
- closing #168 
- closing #169  
- closing #167 (after @ckyriazis did much of this)
- and maybe #159 - I said something about this, but maybe we shold elaborate more?

The diff: [popSimManu-diffmaster.pdf](https://github.com/popsim-consortium/manuscript/files/4503997/popSimManu-diffmaster.pdf)

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -647,25 +677,51 @@ \section*{Discussion}
...
 population genomics community.
 
+One of our goals is to engage research communities studying other taxa,
+so as to expand the resource to many more species.
+Although we have included demographic models and recombination maps,
+there are many biological processes that we do not model.
+Some of these additions that we are enthusiastic to add are:
+selection (including distributions of fitness effects, maps of functional elements, and single sweeps),
+gene conversion, mutation models (rate heterogeneity),
+ancient samples,
```
Comment (grahamgower) [inline popSimManu.tex]: We already have support for sampling in the past. Does this refer to something else?

**Comment by grahamgower** [comment]
Looks good! FYI, the latex diff doesn't seem to be attached.

**Comment by petrelharp** [comment]
Whoops - edited to attach the diff. =)

File: popSimManu.tex (hunk 1 above)
Comment (petrelharp) [reply popSimManu.tex]: Whoops - that must have been written before we added it. Removed!

**Comment by jradrion** [comment]
@petrelharp Should I wait for this PR to be merged before submitting my own PR with the SLiM validation figs?

**Comment by petrelharp** [comment]
If it's holding you up, let's merge - but I'm waiting to see if others would like to weigh in here. I'll merge at the end of today unless I hear otherwise.

**Comment by ckyriazis** [comment]
These changes look great to me!

**Comment by jradrion** [comment]
No worries, and no need to rush a merge before everyone has the chance to take a look!

**Pull request merged**